src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -31,6 +31,7 @@ function getPaymentMethodTitle(
 
     return method => {
         const methodName = getPaymentMethodName(language)(method);
+        const { payPalCreditProductBrandName } = method.initializationData;
```

Comment:
Could we use `getPaymentMethodDisplayName()` to get the title content?

Reply:
Also do we need to check if `payPalCreditProductBrandName` is present on initializationData or not?

Reply:
@bc-peng I've checked getPaymentMethodDisplayName() - it returns Paypal instead of Pay Later, it is do not recognise different paypal branding names for different countries. @bc-fetisov created this payPalCreditProductBrandName value, i believe, because we didn't have such division before.

@animesh1987 we are checking if it present here:
` [PaymentMethodId.PaypalCommerceCredit]: {
                logoUrl: cdnPath('/img/payment-providers/paypal_commerce_logo_letter.svg'),
                titleText: payPalCreditProductBrandName ? payPalCreditProductBrandName.credit : 'Pay Later',
            },`
If we have no payPalCreditProductBrandName- we just passing Pay Later

Reply:
You might [pass the `method` to `getPaymentMethodDisplayName()`](https://github.com/bigcommerce/checkout-js/blob/6b843c7c90122cb74c1dc77a420ab9bf48a1b13e/src/app/payment/paymentMethod/getPaymentMethodDisplayName.tsx#L13) and get the name back like `AdyenV2`.

Reply:
Yes, i did getPaymentMethodDisplayName(language)(method) and it returns Paypal in our case

Reply:
I mean you could put `const { payPalCreditProductBrandName } = method.initializationData;` inside `getPaymentMethodDisplayName(language)(method)`. Then it will match `checkout-js`'s previous logic.

Reply:
Moved logic to getPaymentMethodDisplayName